lib/registration.pm

```diff
@@ -273,7 +279,7 @@ sub register_addons_cmd {
         my $name = get_addon_fullname($addon);
         if (length $name) {
             record_info($name, "Register $name");
-            if (grep(/$name/, @SLE12_MODULES) and is_sle('<15')) {
+            if ((grep(/$name/, @SLE12_MODULES) and is_sle('<15')) || grep(/$name/, @SLE15_MODULES)) {
```

Comment:
It is better to add condition of is_sle('15+') for SLE15_MODULES in case some modules will be in version 12 and 15.